packages/core/src/components/toolbar/test/standalone/e2e.js

```diff
@@ -1,9 +1,19 @@
+'use strict';
 
-  const { register, navigate } = require('../../../../../scripts/e2e');
+const { By, until } = require('selenium-webdriver');
+const { register, Page, platforms } = require('../../../../../scripts/e2e');
 
-  describe('toolbar/standalone', () => {
-
-    register('should init', navigate('http://localhost:3333/src/components/toolbar/test/standalone'));
+class E2ETestPage extends Page {
```

Comment:
Instead of extending `Page` in basic navigation tests like these, I think we can just do something like this:
```js
const { register, navigate, platforms } = require('../../../../scripts/e2e');

describe('toolbar/standalone', () => {
  platforms.forEach(platform => {
    register('should init', navigate(`http://localhost:3333/src/components/toolbar/test/standalone?ionicplatform=${platform}`))
  })
})
```

Reply:
One advantage of doing it this way, though, is that it makes it more like the more complex tests, meaning it would be easy to create a starter template (or generator) that could be used for any e2e test.